zebra-chain/src/block/height.rs

```diff
@@ -24,6 +25,14 @@ pub mod json_conversion;
 #[derive(Copy, Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub struct Height(pub u32);
 
+#[derive(Error, Debug)]
+pub enum HeightError {
+    #[error("The resulting height would ovferflow Height::MAX.")]
```

Comment:
```suggestion
    #[error("The resulting height would overflow Height::MAX.")]
```

Reply:
This was a handy typo because it made the length of the line match the one below for underflows. Goodbye, typo.